tests/testData.h

```diff
@@ -703,6 +705,46 @@ const Species &benzeneSpecies()
     return benzene_;
 }
 
+// Return Node-Graph water test species
+Species *createWater(Graph *parentGraph)
+{
+    const auto name = "Water";
+
+    // Add water species node
+    auto speciesNode = std::make_unique<SpeciesNode>(parentGraph);
+    auto species = &(speciesNode.get()->species());
+    parentGraph->addNode(std::move(speciesNode), name);
+
+    // Set up water species and atom types
+    species->clear();
+    species->setName(name);
+
+    auto oW = std::make_shared<AtomType>("OW");
+    oW->setZ(Elements::Element::O);
+    species->addAtom(Elements::Element::O, Vector3{0.156743, -0.152502, 0.317676}, -0.82, oW);
+
+    auto hW1 = std::make_shared<AtomType>("HW");
+    hW1->setZ(Elements::Element::H);
+    species->addAtom(Elements::Element::H, Vector3{-0.257130, 0.637822, 0.198948}, 0.41, hW1);
+
+    auto hW2 = std::make_shared<AtomType>("HW");
+    hW2->setZ(Elements::Element::H);
+    species->addAtom(Elements::Element::H, Vector3{0.100387, -0.485320, -0.516624}, 0.41, hW2);
+
+    species->simplifyAtomTypes();
+
+    assert(species->atom(0).Z() == Elements::Element::O);
+    assert(species->atom(1).Z() == Elements::Element::H);
+    assert(species->atom(2).Z() == Elements::Element::H);
+
+    // Apply intramolecular terms
+    species->addBond(0, 1).setInteractionFormAndParameters(BondFunctions::Form::Harmonic, "k=4431.53 eq=1.012");
+    species->addBond(0, 2).setInteractionFormAndParameters(BondFunctions::Form::Harmonic, "k=4431.53 eq=1.012");
```

Comment:
Just reverting to SPC/Fw default values:
```suggestion
    species->addBond(0, 1).setInteractionFormAndParameters(BondFunctions::Form::Harmonic, "k=4431.53 eq=1.0");
    species->addBond(0, 2).setInteractionFormAndParameters(BondFunctions::Form::Harmonic, "k=4431.53 eq=1.0");
```